closed/get_j9_source.sh

```diff
@@ -109,6 +119,31 @@ for i in "$@" ; do
 	esac
 done
 
+# parse vendors
+for i in "${!vendors[@]}"; do
+	# convert comma separated string - e.g. git@github.com:vendor/avendor.git,master,5fe1078 - to array
+	vendor=(`echo ${vendors[$i]} | sed 's/,/\n/g'`)
+
+	# extract the name of the repository from the URL
+	vendor_project=(`echo ${vendor[0]} | sed -e 's/\// /g' | sed -e 's/\.git//g' |  awk '{print $NF}'`)
```

Comment:
This doesn't seem robust as there could be multiple repos with the same name at different locations.
i.e. eclipse/openj9, ibmextensions/openj9

Reply:
I don't think there shouldn't be copying over of git repos.  This makes it difficult from a development standpoint determining what is noise and what it is that they are actually working on.

Reply:
Copying the vendor source over openj9 at this stage (before compile) would mess up the developers workspace making difficult the vendor source development.